HUBSPOT_TO_S3/php/.env.default

```diff
@@ -6,4 +6,4 @@ PAN_CFG_MAKE_COMPANY=true
 PAN_CTX_RUN_MODE=normal
 PAN_SEC_AWS_SECRET_ACCESS_KEY=client_aws_secret_key
 PAN_SEC_AWS_ACCESS_KEY_ID=client_aws_key_id
-PAN_SEC_HUBSPOT_BASIC_API_KEY=client_hubspot_api_key
+PAN_SEC_HUBSPOT_ACCESS_TOKEN=client_hubspot_api_key
```

Comment:
If you create this integration using hubspots ouath connector, and oauth dance your way to getting an access token i think this will work as is on the platform.

Another option is to create new hubspot connector that is basic and accepts an access_token on connection.